source/objects/RenderTarget.cpp

```diff
@@ -209,6 +214,119 @@ Pixel RenderTarget::SampleBilinear(float u, float v, uint8_t mip) const
 	};
 }
 
+bool RenderTarget::Load(const char* filename) {
+	if (!IsValid()) return false;
+
+	int width = 0;
+	int height = 0;
+	int channels = CHANNELS[static_cast<uint8_t>(GetFormat())];
+	
+	size_t stride = STRIDES[static_cast<uint8_t>(GetFormat())];
+
+	if (stride == sizeof(uint8_t)) {
+		// Load it as an unsigned char array
+		stbi_uc* imageBuffer = stbi_load(filename, &width, &height, nullptr, channels);
+		if (!imageBuffer) {
+			return false;
+		}
+
+		bool resized = Resize(width, height);
+		if (!resized) {
+			stbi_image_free(imageBuffer);
+			return false;
+		}
+
+		memcpy(GetRawData(), imageBuffer, stride * width * height * channels);
+		GenerateMIPs();
+		stbi_image_free(imageBuffer);
+
+		return true;
+	}
+	else if (stride == sizeof(float)) {
```

Comment:
You don't need to duplicate *all* of the above code, just change the load call as needed and use `uint8_t` for the image buffer either way